src/Microsoft.Identity.Abstractions/TokenAcquisition/AcquireTokenOptions.cs

```diff
@@ -90,6 +91,31 @@ public AcquireTokenOptions(AcquireTokenOptions other)
         /// </summary>
         public string? PopClaim { get; set; }
 
+        /// <summary>
+        /// When <see cref="AcquireTokenOptions"/> is set using <see cref="ManagedIdentityOptions"/>, it specifies the ClientId, 
+        /// ObjectId, and HostUserId of the Azure user-assigned managed identity used to aquire an app token instead of using
+        /// client credentials. This requires the application to be deployed on Azure, the managed identity to be configured, and
+        /// the workload identity federation containing the managed identity is declared in the application registration.
```

Comment:
```suggestion
        /// 
```

This is a different scenario,